cli/README.md

```diff
@@ -63,6 +63,13 @@ $ kubectl kadalu storage-add storage-pool-1 \
     --pv-reclaim-policy=archive
 ```
 
+To re-create storage-pool from existing volume ID,
+
+```
+$ kubectl kadalu storage-add storage-pool-1 \
```

Comment:
Partial syntax will confuse users. Use a complete one including the device/path/pvc.

Reply:
Done